File: DTO/Base/GetListPagingRequest.cs
namespace DTO.Base
{
    public class GetListPagingRequest
    {
        public string? Search { get; set; }
        public DateTime FromDate { get; set; }
        public DateTime ToDate { get; set; }
        public int Offset { get; set; }
        public int Limit { get; set; } = 10;
        public string? Order { get; set; }
        public string? Sort { get; set; }
    }
}

File: GUI/src/app/models/base/getListPagingRequest.ts
export interface GetListPagingRequest {
  search: string | '';
  fromDate: Date | null;
  toDate: Date | null;
  offset: number | 0;
  limit: number | 10;
  order: string | null;
  sort: string | null;
}

export function CreateDefaultGetListPagingRequest(): GetListPagingRequest {
  return {
    search: '',
    fromDate: null,
    toDate: null,
    offset: 0,
    limit: 10,
    order: null,
    sort: null,
  }
}
